src/scenes/ModulesRouter/ModuleEventView.tsx

```diff
@@ -111,58 +69,85 @@ const ModuleEventView = () => {
+                        window.open(event.eventURL, '_blank', 'noopener noreferrer');
+                      }
+                    }}
+                  />
                 </Box>
-              ))}
 
-              {eventIsLiveSession && <Box dangerouslySetInnerHTML={{ __html: mockEventData?.description || '' }} />}
+                {event?.links?.map((link, index) => (
+                  <Box key={index} marginBottom={theme.spacing(2)}>
+                    <Link
+                      fontSize='16px'
+                      onClick={() => {
+                        if (link.url) {
+                          window.open(link.url, '_blank', 'noopener noreferrer');
+                        }
+                      }}
+                    >
+                      {link.label}
+                    </Link>
+                  </Box>
+                ))}
+              </Grid>
+
+              <Grid item xs={6} sx={{ flexGrow: 1, padding: `${theme.spacing(1)} ${theme.spacing(3)}` }}>
+                <Box
+                  dangerouslySetInnerHTML={{ __html: event?.callDescription || '' }}
+                  sx={{ backgroundColor: theme.palette.TwClrBgSecondary, borderRadius: '8px', padding: '8px 16px' }}
+                />
+              </Grid>
+            </Grid>
 
-              {mockEventData?.additionalLinks?.length && (
-                <>
-                  <hr
-                    style={{
-                      border: 'none',
-                      borderTop: `1px solid ${theme.palette.TwClrBrdrTertiary}`,
-                      marginBottom: theme.spacing(2),
-                    }}
-                  />
+            <Grid container spacing={theme.spacing(1)}>
+              <Grid item xs sx={{ flexGrow: 1, padding: `${theme.spacing(1)} ${theme.spacing(3)}` }}>
+                <hr
+                  style={{
+                    border: 'none',
+                    borderTop: `1px solid ${theme.palette.TwClrBrdrTertiary}`,
+                    margin: `${theme.spacing(3)} 0`,
+                  }}
+                />
 
-                  {mockEventData?.additionalLinks?.map((link, index) => (
-                    <Box key={index} marginBottom={theme.spacing(2)}>
-                      <Link fontSize='16px' to='#'>
-                        {link.label}
-                      </Link>
-                    </Box>
-                  ))}
-                </>
-              )}
+                <Box dangerouslySetInnerHTML={{ __html: event?.description || '' }} />
+
+                {event?.additionalLinks?.length && (
+                  <>
+                    {event?.additionalLinks?.map((link, index) => (
+                      <Box key={index} marginBottom={theme.spacing(2)}>
+                        <Link
+                          fontSize='16px'
+                          onClick={() => {
+                            if (link.url) {
+                              window.open(link.url, '_blank', 'noopener noreferrer');
```

Comment:
Alternatively use the `to=` and `target=` properties of `Link`? Not sure if the `rel=` is auto added, maybe we could update Link to do that.

Reply:
So these URLs are external and it seemed like `Link` was specific for internal app routes, but I'll try it out and will update if `Link` handles external URLs.